types/multiavatar__multiavatar/tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
your module should work anyway, please remove this. 
If anything, skip my comment about removing 'node' dependency, just to properly test your inclusions,
but skip this flag. It's for consuming native package, not used on DT definitions itself

Reply:
Removed. Please do explain the node dependency comment though (in that thread). I'm interested in what you meant by that.